Solar-System

```diff
@@ -0,0 +1,97 @@
+# Brittany Jones
+# Ada Solar system
+# Last edited 2/9/18
+
+# Create a SolarSystem class with an @planets instance variable.
+# Create an initialize method which should take a collection of planet names and store them in an @planets instance variable.
+
+# Create a class with a hash of planets and their attributes.
+# Give each planet a year_length attribute which is the length of time the planet takes to go around its star.
+# Give each planet a distance_from_the_sun attribute
+
+class Planet
+  # Create reader methods to give a user access to read the instance variables.
+attr_accessor :planet_hash
+def initialize(name, color, moons, radius, distance_from_the_sun, year_length, fun_fact)
+@planet_hash = {
+"Name" =>  name,
+"Color" =>  color,
+"Moons" => moons,
+"Radius"  => radius,
+"Distance from the Sun" => distance_from_the_sun,
+"Year Length" => year_length,
+"Fun Fact" => fun_fact
+    }
+  end
+end
+
+#Create a SolarSystem class with an @planets instance variable.
+#Make your SolarSystem class take an array of Planets instead of hashes. class will take an array of hashes, instead of hashes.
+class SolarSystem
+attr_accessor :planets
+def initialize
+@planets = planets
+@planets = []
```

Comment:
You are blanking out `@planets` so anything sent into initialize is erased.

Also your `initialize` method isn't getting sent a parameter.  

You should have:

```ruby
def initialize(planets)
  @planets = planets
end
```